adapter/internal/egress/retry_writer_test.go

```diff
@@ -245,6 +250,34 @@ func (s *spyLogClient) EmitLog(message string, opts ...loggregator.EmitLogOption
 	s.sourceInstance = env.GetTags()["source_instance"].GetText()
 }
 
+func (s *spyLogClient) Message() string {
```

Comment:
In my view, it's important to make methods available only on the spy private. Otherwise, the public method signals to a reader that it is part of the injected interface.